lib/widgets/directions_drawer.dart

```diff
@@ -1,51 +1,420 @@
+                                          color: constants.offWhiteColor,
+                                          border: Border(
+                                            bottom: BorderSide(
+                                                width: 1.0,
+                                                color:
+                                                    constants.lightGreyColor),
+                                          ),
+                                        ),
+                                        child: ListTile(
+                                          contentPadding: EdgeInsets.only(
+                                              left: SizeConfig
+                                                      .safeBlockHorizontal *
+                                                  5.0,
+                                              right: SizeConfig
+                                                      .safeBlockHorizontal *
+                                                  5.0),
+                                          leading: Icon(
+                                              Icons.subdirectory_arrow_right),
+                                          title: Text(
+                                            "$key",
+                                            style: GoogleFonts.raleway(
+                                              fontSize: 17.0,
+                                              fontWeight: FontWeight.w600,
+                                              color: constants.blackColor,
+                                            ),
+                                          ),
+                                          subtitle: Text(
+                                            "Time: " +
+                                                "${itinerary.data[key].keys.toString().replaceAll(RegExp(r"\(|\)"), "")}" +
+                                                "  " +
+                                                "Distance: " +
+                                                "${itinerary.data[key].values.toString().replaceAll(RegExp(r"\(|\)"), "")}",
+                                            style: GoogleFonts.raleway(
+                                              fontSize: 15.0,
+                                              fontWeight: FontWeight.w400,
+                                              color: constants.blackColor,
+                                            ),
+                                          ),
+                                        ),
+                                      );
+                                    },
+                                  ),
+                                );
+                            }
+                          }),
+                    ), // list of directions
+                    ShuttleWidget(),
+                  ],
+                )));
+
+    if (Provider.of<MapData>(context).itinerary == null) {
```

Comment:
What's the purpose of this? Why is it here?

Reply:
Check 7c8a951